### PR TITLE
[feat] Added Screens and auth

### PR DESCRIPTION
**Pull request by rishichourasia**
image
image

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,13 +1,24 @@
...
+} from "./components/export";
 
 function App() {
 	return (
 		<>
 			<Navbar />
-			<div className="main">
-				<Sidebar />
-			</div>
+			<Routes>
```
Comment (Chiragmodi01) [inline src/App.js:15]: `Sidebar` comp also should be here, if you're using grids. Placing it on every page is not optimal solution. Cause `sidebar` isn't changing, while the page is!

File: src/backend/controllers/AuthController.js [hunk 2]
```diff
@@ -15,42 +15,42 @@ const sign = require("jwt-encode");
  * */
 
 export const signupHandler = function (schema, request) {
```
Comment (Chiragmodi01) [inline src/backend/controllers/AuthController.js:17]: Looks like you beautified controller files, unnecessarily it added indents and now showing in changed file in pr!

File: src/components/Login/Login.jsx [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+			});
+			localStorage.setItem("token", response.data.encodedToken);
+			setToken(response.data.encodedToken);
+			console.log(response.data.encodedToken ? "Logged in" : "Error");
+			navigate("/");
+		} catch (err) {
+			console.log(err);
+		}
+	};
+
```
Comment (Chiragmodi01) [inline src/components/Login/Login.jsx:25]: I'd suggest you create separate 'services' files to handle Api fetch and calls.

File: src/components/Login/Login.jsx [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+					placeholder="Enter Email"
+				/>
+			</div>
+			<div className="input-div">
+				<label>Password</label>
+				<input className="input" type="password" placeholder="Enter Password" />
+			</div>
+			<label className="label">
+				<input type="checkbox" name="label" id="" /> Remember me
+			</label>
```
Comment (Chiragmodi01) [inline src/components/Login/Login.jsx:44]: Add `id` in input and `htmlFor` in label. It'll take label clickable and function better as well.

File: src/components/Login/Login.jsx [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+			<button
+				className="btn btn-primary"
+				onClick={() => {
+					handleLogin();
+				}}
+			>
+				Login
```
Comment (Chiragmodi01) [inline src/components/Login/Login.jsx:51]: Beautifying code seem to made it messy, adding indents and unnecessary spaces.

File: src/components/Login/login.css [hunk 6]
```diff
@@ -0,0 +1,51 @@
+@import url("/src/styles/main.css");
```
Comment (Chiragmodi01) [inline src/components/Login/login.css:1]: Would suggest not importing any css other files in existing css. It'll show clashes while overriding. (you can see it in elements).

File: src/components/Navbar/Navbar.jsx [hunk 7]
```diff
@@ -1,27 +1,31 @@
...
-							className="logo-svg"
-							src="/assets/logo/logo simple dark.svg"
-							alt="logo-svg"
-						/>
-					</div>
+					<NavLink to="/">
+						<div className="menu-btn">
+							<img
+								className="logo-svg"
+								src="/assets/logo/logo simple dark.svg"
```
Comment (Chiragmodi01) [inline src/components/Navbar/Navbar.jsx:14]: export all assets from a single file and use it, importing as `{ }` , wherever needed.

File: src/components/Notespage/NotesPage.jsx [hunk 8]
```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import { useEffect } from "react";
+import { useAuth } from "../../context/auth-context";
+import { useNotes } from "../../context/note-context";
```
Comment (Chiragmodi01) [inline src/components/Notespage/NotesPage.jsx:4]: bunch import from same source. `{ useAuth, useNotes }`

File: src/components/Notespage/notespage.css [hunk 9]
```diff
@@ -0,0 +1,62 @@
...
+
+.inputdiv {
+	display: flex;
+	justify-content: center;
+	padding-top: 1rem;
+}
+.inputbox input {
+	border: none;
+	font-size: 1.1rem;
+	margin: 5px 10px 5px 10px;
```
Comment (Chiragmodi01) [inline src/components/Notespage/notespage.css:17]: trim it to `margin: 5px 10px;`

File: src/components/Sidebar/sidebar.css [hunk 10]
```diff
@@ -26,6 +26,6 @@
 	font-size: 1.2rem;
 }
 
-.sidebar-label:hover {
+/* .sidebar-label:hover {
 	background-color: var(--light-grey-color);
-}
+} */
```
Comment (Chiragmodi01) [inline src/components/Sidebar/sidebar.css:31]: remove commented code

File: src/components/Signup/Signup.jsx [hunk 11]
```diff
@@ -0,0 +1,34 @@
...
+					autoComplete="nope"
+					placeholder="Enter Email"
+				/>
+			</div>
+			<div className="input-div">
+				<label>Password</label>
+				<input className="input" type="password" placeholder="Enter Password" />
+			</div>
+			<label className="label">
+				<input type="checkbox" name="label" id="" /> Accept all terms and
```
Comment (Chiragmodi01) [inline src/components/Signup/Signup.jsx:22]: same comment here for, `id` and `for` in label

**Review by Chiragmodi01 — COMMENTED** [review]
- Beautifying files sometimes mess up, makes it hard to read. So make the code neat yourself if possible.
- Give `notecard` minimum and maximum height and width. If exceeds max width, wrap it.
- Restrict user from adding empty note.
- Add margin bottom to note's title. (preferable)
- Creating a separate button for signup doesn't make sense, just adding it in login would suffice.

Added few sanity comments.
Overall UI seems good.

File: src/App.js (hunk 1 above)
Comment (rishichourasia) [reply src/App.js:15]: yeah i know , but i don't want them in my signup/login page thats is why had to import it to every comp which needs it.

**Pull request merged**